docs/api/expect.md

```diff
@@ -5,6 +5,22 @@ title: Expect
 
 When you're writing tests, you often need to check that values meet certain conditions. `expect` gives you access to a number of "matchers" that let you validate different things on the `browser` or an `element` object.
 
+## Default Options
```

Comment:
Agreeing with @christian-bromann's statement below, it'd be worth mentioning,
- the provided default options
- that these can be configured in the wdio.conf
- for specific tests, the can be overridden 
    - does a note need to be added this should be reset or all other tests will use the overridden values?